crates/mako/src/generate/chunk_pot/str_impl.rs

```diff
@@ -112,12 +112,16 @@ pub(super) fn render_normal_js_chunk(
 ) -> Result<ChunkFile> {
     let (content_buf, source_map_buf) = {
         let pot = chunk_pot;
+
+        // to avoid ' or " been included in chunk_loading_global
+        let safe_prop =
+            serde_json::from_str::<serde_json::Value>(&context.config.output.chunk_loading_global)
```

Comment:
try this
```rust
    let my_string = String::from(r#"Hello, " world!"#);
    let serialized = serde_json::to_string(&my_string).unwrap();
```


https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=c77dda62d9668b6d3b1e30af52180bee

Reply:
nice